app/models/ability.rb

```diff
@@ -49,11 +49,11 @@ def initialize(user)
       user.admin? || user == attendance.user
     end
 
+    #todo add mailing controller and view for users in their namespace, where applicable
```

Comment:
I'd like to move the user's own mailings in their namespace/dashboard.
Also, I'll have to check if mailings will pop up in the activity page; and if we want them overthere in the first place. To be continued.